client/landing/stepper/declarative-flow/internals/steps-repository/importer/index.tsx

```diff
@@ -83,7 +83,7 @@ export function withImporterWrapper( Importer: ImporterCompType ) {
 			}
 		}, [ siteId, site ] );
 
-		if ( ! importer ) {
+		if ( ! importer || ! fromSiteData ) {
```

Comment:
We can't have this here because this is where we might have users come directly. For example, we have our progress screen and plans page here where users might come directly if they dropped off the flow earlier. `http://calypso.localhost:3000/setup/import-focused/importerWordpress?siteSlug=${target_site}&from=${source_site}&option=everything`. We'll fetch the site data if there's no data fetched yet here. If we add the check here, it'll always be redirected to the site address URL input page before it tries to fetch the site data.

Reply:
Right. What's the appropriate place for checking, then? Right before usage? What should we do if there is no `fromSiteData`? Is this data optional?